### ISSUE TITLE
Files created with wrong namespace

### ISSUE THREAD

**Issue by nilaxann65** [microsoft/vscode-dotnettools#1145]
Type: <b>Bug</b>

1. Access to an existent project or create a new one from the Solution Explorer
2. Navigate to a nested folder
3. Create a new File with any Template 

On this step the new file will be created on the selected folder, but with the root namespace, ignoring the parent folders when naming the namespace

Extension version: 1.6.8
VS Code version: Code 1.89.1 (dc96b837cf6bb4af9cd736aa3af08cf8279f7685, 2024-05-07T05:13:33.891Z)
OS version: Windows_NT x64 10.0.22631
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|12th Gen Intel(R) Core(TM) i7-1255U (12 x 2611)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|31.71GB (15.96GB free)|
|Process Argv|. --crash-reporter-id 7289693d-a32e-45d9-83bf-07973d32a0ce|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492:30256859
vscod805cf:30301675
binariesv615:30325510
vsaa593cf:30376535
py29gd2263:31024239
c4g48928:30535728
azure-dev_surveyonecf:30548226
a9j8j154:30646983
962ge761:30959799
pythongtdpath:30769146
welcomedialog:30910333
pythonidxpt:30866567
pythonnoceb:30805159
asynctok:30898717
pythontestfixt:30902429
pythonregdiag2:30936856
pythonmypyd1:30879173
pythoncet0:30885854
h48ei257:31000450
pythontbext0:30879054
accentitlementst:30995554
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
chatpanelt:31048053
dsvsc021:30996838
724cj586:31013169
pythoncenvptcf:31049071
fchga941:31046352
pythonprt:31056678
dwnewjupytercf:31046870
26j00206:31048877

```

</details>

**Comment by WithLithum** [comment]
Can reproduce this issue on Ubuntu, snap version, with no A/B.

**Comment by smitpatel** [comment]
Fixed in version 1.9.55
